src/first-print/calibration.md

```diff
@@ -34,3 +44,195 @@ print. Too little will result in gaps and poor inter-layer adhesion.
+
+- Get the stl File
+- Put it in Slic3r
+- Go to Print Settings. Choose the following Settings:
+``` cfg
+General:
+- Layer height: 0.2 mm
+- Perimeters: 1
+- Solid Layers Top: 0, Bottom: 3
+
+Infill:
+- Fill density: 0%
+
+Speed:
+- Perimeters: 40 mm/s 
+``` 
+Start printing the object with the max temperature of the supplyer. Whenever you see a indentation (every 5 mm), lower your temperature by 5°C.  
+Stop the print when your nozzle can't print out enough Filament (as whown in the Example below) or the print quality decreases again
+Here are two exaples:
+![white heat tower](http://thingiverse-production-new.s3.amazonaws.com/renders/96/e3/5c/55/3d/491730c03f3cd431ee343f3425c278dc_display_large.jpg)
+![brown heat tower](http://thingiverse-production-new.s3.amazonaws.com/renders/66/5b/99/36/76/cbd49ff4089968bef2847d5542caef73_display_large.jpg)
+
+The area with the best layer adhesion AND best visuals is your perfect temperature for this particular speed and material. Usually the higher the Temparature the better the adhesion so you can take the highest, good looking Temperature.
+
+If you don't reach your perfect temperature with one heatcube you can print another one starting with the last temperature.
+
+You may also print another heattower, start by 5 degrees over your "perfect" temperature and decrease it by 1° steps. then choose a temperatue 2-3 degrees under the "too hot" temperature
+
+In the examples you can also see why its important to do this with every material/Manufacturer: The filaments tend to have different ideal temperatures which ma even vary by colors from the same manufacturer.
+
+##### Extrusion Multiplier
+Print out the general purpose 20mm Cube with the following Settings:
+``` cfg
+- vase mode (single outline, no infill, no top or bottom layers)
+- extrusion multiplier: 1
+- extrusion width = nozzle width
+```
+- Measure the thickness of all 4 walls with a calipher and get an average of the 3 smallest values.
+- Change your extrusion multiplier:  new multiplier = old multiplier x (extrusion width / average measurement)
+- Repeat until you are happy
+
+Tips:
+- Make sure you measure from the top as the bottom might be flatted out from the Bed.
+- if you have big variations it is a good idea to measure the side that is extruded last as the extrusion will have mostly stabilised by then
+- Try to measure the smallest number of layers possible from top as the slightest vertical misalignment of your layers will increase the measured value. Ideally a single layer would be best but very difficult with regular calipers.
+- Make sure you dont apply to much pressure with your caliphers (Make sure the measuring device is not biting into the filament during measurement.)
+
+
+**Do not forget to save Your calibration settings.**
+
+### Advanced tuning
+#### Requirements
+- You are able to print in reasonable quality after your "Basic Tuning"
+- Switch to expert mode
```

Comment:
Expert mode is no longer a thing, so this can be removed.

Reply:
what does that mean exactly? I haven't been in the scene for a wile. does it mean you only have the "simple" settings or you only have the "expert" settings?

Reply:
We don't have simple or expert mode. We are all experts :D